pages/en/irma-start.md

```diff
@@ -178,22 +176,22 @@ possibilities.
   re-register from the start.
 
 * At the MyIRMA webpage you can also manage which email address, if
-  any, is associated with your account.
+  any, is associated with your IRMA app.
```

Comment:
I'm not sure if this is clear enough that this is per app installation then. We could make this more explicit, e.g.:
`IRMA app. Since you can install IRMA independently on multiple devices, the IRMA installation on each device is given an IRMA app ID.`
Or something similar.